HotTips/TipOfTheDayWindow.xaml.cs

```diff
@@ -226,6 +231,21 @@ private void ShowTipContent(TipInfo nextTip)
             string tipContentString = ReadStringFromFile(contentFilePath);
             // Update the value in the model (which should be reflected in the view through binding)
             _tipViewModel.TipContent = tipContentString;
+
+            List<TipHistoryInfo> tipHistory = _tipHistoryManager.GetTipHistory();
+            TipHistoryInfo historyData = tipHistory.Find(a => a.globalTipId.Equals(nextTip.globalTipId));
+            if (historyData != null)
+            {
+                if (historyData.tipLikeStatus.Equals(TipLikeEnum.LIKE))
+                {
+                    PopulateLikeFilledImage();
+                }
+
+                if (historyData.tipLikeStatus.Equals(TipLikeEnum.DISLIKE))
+                {
+                    PopulateDislikeFilledImage();
```

Comment:
This relies on having previously called PopulateDefaults.
It would be better if there were methods for "ShowLikeVote" and "ShowDislikeVote" and "ShowNoVote". So the populate logic is all contained in one place.